protected/humhub/modules/space/notifications/SpaceCreated.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2018 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\space\notifications;
+
+use humhub\modules\notification\components\BaseNotification;
+use Yii;
+use yii\helpers\Html;
+
+/**
+ * @since 1.16
+ */
+class SpaceCreated extends BaseNotification
+{
+    /**
+     * @inheritdoc
+     */
+    public $moduleId = 'space';
+
+    /**
+     * @inheritdoc
+     */
+    public $viewName = 'spaceCreated';
+
+    /**
+     * @inheritdoc
+     */
+    public function category()
+    {
+        return new SpaceCreatedNotificationCategory();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getMailSubject()
+    {
+        return $this->getInfoText($this->originator->displayName, $this->source->displayName);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function html()
+    {
+        return $this->getInfoText(
+            Html::tag('strong', Html::encode($this->originator->displayName)),
+            Html::tag('strong', Html::encode($this->source->name)),
+        );
+    }
+
+    private function getInfoText($displayName, $spaceName)
+    {
+        return Yii::t('SpaceModule.notification', '{displayName} created a the new space {spaceName}', [
```

Comment:
@marc-farre Uppercase the word "Space"

Reply:
@luke- Commit https://github.com/humhub/humhub/pull/7057/commits/c0a12dc533433dbfb95243711c815b4f355b0398